Tests/scripts/update_id_set.py

```diff
@@ -305,12 +321,12 @@ def process_integration(file_path):
     """
     res = []
     if os.path.isfile(file_path):
-        if re.match(INTEGRATION_REGEX, file_path, re.IGNORECASE) or \
-                re.match(BETA_INTEGRATION_REGEX, file_path, re.IGNORECASE):
+        if checked_type(file_path, (INTEGRATION_REGEX, BETA_INTEGRATION_REGEX, PACKS_INTEGRATION_YML_REGEX,
```

Comment:
`PACKS_INTEGRATION_YML_REGEX` should not be here. it should be in the else part as it relates to the integration package (dir)